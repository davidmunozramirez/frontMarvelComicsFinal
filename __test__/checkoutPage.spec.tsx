import {render} from "@testing-library/react";
import CheckoutComic from "dh-marvel/pages/checkout/[id].page";

describe('CheckoutComic', () => {
  it('should render correctly', () => {
    const { asFragment } = render(<CheckoutComic />);
    expect(asFragment()).toMatchSnapshot();
  })
})