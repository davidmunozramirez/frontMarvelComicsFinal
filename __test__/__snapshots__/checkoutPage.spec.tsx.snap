// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CheckoutComic should render correctly 1`] = `<DocumentFragment />`;
